frameworks/Go/sprapp/README.md

```diff
@@ -0,0 +1,5 @@
+SPRAPP https://github.com/cloudxaas/sprapp/
```

Comment:
Why isn't this called hertz-prefork? @hiqsociety  Should we add a new hertz-prefork ?

Reply:
cc @nbrady-techempower 
![image](https://github.com/TechEmpower/FrameworkBenchmarks/assets/97824201/55647f72-4996-42e8-b68f-f5dc30a7697d)

Reply:
the idea is like fiber-prefork.
will be using like fiber framework